src/backend/InvenTree/order/test_api.py

```diff
@@ -1919,6 +1919,12 @@ def test_download_csv(self):
 class SalesOrderAllocateTest(OrderTest):
     """Unit tests for allocating stock items against a SalesOrder."""
 
+    @classmethod
+    def setUpTestData(cls):
+        """Init routine for this unit test class."""
+        super().setUpTestData()
+        Part.objects.rebuild()  # Ensure part variant trees are correct
```

Comment:
Is this not an indication that we have an issue in the model create/update setup?

Reply:
Also going to add some unit tests to ensure that the fixture data itself is correct.